tests/testthat/test_reset.R

```diff
@@ -2,5 +2,40 @@ context("DGEobj - tests for reset.R functions")
 
 
 test_that('reset.R: ', {
-    #need an object with levels to test this
+
+    # checking DGEobj without levels
+    expect_error(resetDGEobj(DGEobj),
+                 regexp = "!is.null(attr(dgeObj, \"level\")) is not TRUE",
+                 fixed  = TRUE)
+
+    # checking DGEobj without platformType
+    test_DGEobj <- setAttributes(item = DGEobj, list("level" = "gene"))
+    test_DGEobj <- setAttributes(item = test_DGEobj, list("PlatformType" = NULL))
+    expect_error(resetDGEobj(test_DGEobj),
+                 regexp = "Required attribute \"PlatformType\" is missing.",
+                 fixed  = TRUE)
+
+    # checking DGEobj with wrong counts matirx
+    test_DGEobj <- setAttributes(item = test_DGEobj, list("PlatformType" = "RNA-Seq"))
+    expect_error(resetDGEobj(test_DGEobj),
+                 regexp = "`%in%`(x = itemName, table = names(dgeObj)) is not TRUE",
+                 fixed  = TRUE)
+
+    # checking DGEobj with wrong data
+    names(test_DGEobj) <- c( "counts_orig" , "counts", "design_orig" , "design", "peptideAnnotation_orig")
+    expect_error(resetDGEobj(test_DGEobj),
+                 regexp = "Gene/isoform/exon/protein data not found",
+                 fixed  = TRUE)
+
+    # checking valid object
+    names(test_DGEobj) <- c( "counts_orig" , "counts", "design_orig" , "design", "isoformData_orig")
+    # Expecting warning as current object could not build the GRanges object!
```

Comment:
Rephrase comment 

Warning expected as the current object cannot build the GRanges object.